sklearn/feature_extraction/text.py

```diff
@@ -306,6 +306,16 @@ def _check_vocabulary(self):
         if len(self.vocabulary_) == 0:
             raise ValueError("Vocabulary is empty")
 
+    def _check_ngram_range(self):
+        """Check validity of ngram_range parameter,
+        e.g., ngram_range=(2,1) is invalid
+        """
+        min_n, max_m = self.ngram_range
+        if min_n > max_m:
+            raise ValueError(
+                "Invalid value for ngram_range, "
+                "min_n should not be larger than max_m")
```

Comment:
please print the actual values
```py
(".... ngram_rage=%s "
 "lower boundary larger than the upper boundary")
% ngram_range
```
users won't necessarily know what "max_n" refers to when they get this error.